For libpng and libjpeg Makefile.PL tries to run a short test program to see if
the system libraries can be used. From native builds we know they can so remove
the test.

--- PNG/Makefile.PL.orig	2013-11-18 10:04:04.000000000 +0100
+++ PNG/Makefile.PL	2018-09-14 09:31:08.785202470 +0200
@@ -53,8 +53,7 @@
  $all_libs   = "$libpng_libs $zlib_libs";
 }
 
-if ($Tk::MMtry::VERSION ge '4.007' &&
-    try_run("config/has_png.c",[$all_cflags],[$all_libs]))
+if ($Tk::MMtry::VERSION ge '4.007')
  {
   warn "Using system's -lpng\n";
   Tk::MMutil::TkExtMakefile(
--- JPEG/Makefile.PL.orig	2013-11-18 10:04:04.000000000 +0100
+++ JPEG/Makefile.PL	2018-09-14 09:35:28.041623709 +0200
@@ -6,7 +6,7 @@
 
 use Tk::MMtry;
 
-if ($Tk::MMtry::VERSION ge '4.007' && try_run("config/has_jpeg.c",['-I/usr/local/include'],['-ljpeg']))
+if ($Tk::MMtry::VERSION ge '4.007')
  {
   push(@args, LIBS => ['-ljpeg'],
               INC  => '-I/usr/local/include',
